VideoLocker/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -167,6 +177,20 @@ public void download(DownloadEntry videoData) {
         }
     }
 
+    private boolean hasConnectivity(CourseBaseActivity activity) {
```

Comment:
Will it be a good idea to put it `NetworkUtil`, knowing that it requires a `CourseBaseActivity` type argument ?